hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineManager.java

```diff
@@ -346,6 +347,11 @@ public void triggerPipelineCreation() {
     backgroundPipelineCreator.triggerPipelineCreation();
   }
 
+  @Override
+  public PipelineFactory getPipelineFactory() {
```

Comment:
We dont need to expose this api. We can add a shutdown call for the pipelineFactory in the SCMPipelineManager#close fn.

Reply:
Done